### ISSUE TITLE
[A11y] <amp-img> intrinsic sizer missing alt attribute

### ISSUE THREAD

**Issue by ericcecchi** [ampproject/amphtml#20578]
## What's the issue?

When using `layout="intrinsic"` on `<amp-img>`, the intrinsic sizer image is missing an `alt` attribute. This causes screen-readers to read it as "slash greater (/>)" (i.e. the end of the SVG data URI in the `src` attribute) in Chrome.

Since the sizer image has no semantic value to a screen-reader, it should be treated as a decorative image by including an empty `alt=""` attribute and `aria-hidden="true"` for good measure.

## How do we reproduce the issue?

[JSbin exhibiting issue](https://jsbin.com/muxalik/edit?html,output)

1. Turn on the VoiceOver utility.
2. Use keyboard navigation to navigate through web content.
3. In Chrome, VoiceOver will read the size image as "slash greater (/>)".

## What browsers are affected?

I can reproduce this issue in Chrome using VoiceOver on MacOS. Firefox and Safari seems to ignore the image completely with VoiceOver setting set to only read images with descriptions. It's possible that other screen-readers exhibit this issue, but I have not tried to reproduce it with them.

## Which AMP version is affected?

This has been an issue since `layout="intrinsic"` was introduced.

Edit: The issue originally stated that the issue was present in Firefox; however, this only happens if VoiceOver settings are set to always read images.

**Comment by aghassemi** [comment]
Thanks @ericcecchi for raising this. Fix on its way.

**Comment by ericcecchi** [comment]
Thanks for the quick fix @aghassemi!
